docs/FAQ.adoc

```diff
@@ -346,12 +341,12 @@ Possible causes of this error include an incomplete gzip operation, and hardware
 A brute-force way to recover from this error is to remove the rdiff-backup-data directory.
 However, this will remove all of your past increments.
 A better approach may be to delete the particular file that is causing the problem.
-A command like:\ \ `$ find rdiff-backup-data -type f -name \*.gz -print0 | xargs -0r gzip --test`\ \ will find the failing file.
+A command like: `$ find rdiff-backup-data -type f -name \*.gz -print0 | xargs -0r gzip --test` will find the failing file.
 For more information on this approach, see this mailing list post: https://lists.nongnu.org/archive/html/rdiff-backup-users/2007-11/msg00008.html.
 
 == What does "`AssertionError: Bad index order`" mean?
 
-If rdiff-backup fails with the message "`AssertionError: Bad index order`," it could be because the files in a directory have changed while rdiff-backup is running.
+If rdiff-backup fails with the message `AssertionError: Bad index order`," it could be because the files in a directory have changed while rdiff-backup is running.
```

Comment:
```suggestion
If rdiff-backup fails with the message `AssertionError: Bad index order`, it could be because the files in a directory have changed while rdiff-backup is running.
```